openfl/interface/interactive_api/experiment.py

```diff
@@ -233,19 +232,19 @@ def start(self, *, model_provider, task_keeper, data_loader,
     def define_task_assigner(self, task_keeper, rounds_to_train):
         """Define task assigner by registered tasks."""
         tasks = task_keeper.get_registered_tasks()
-        self.train_task_exist = False
+        train_task_exist = False
         self.validation_task_exist = False
         for task in tasks.values():
             if task.task_type == 'train':
-                self.train_task_exist = True
+                train_task_exist = True
             if task.task_type == 'validate':
                 self.validation_task_exist = True
```

Comment:
Maybe make sense to rewrite `train_task_exist` and `self.validate_task_exist` as properties `self.is_train_task_exist` and `self.is_validate_task_exist`.

Reply:
They were renamed, but I keep they scope and didn't make them as properties. It will be easy to fix if needed.